src/layout/CustomButton/CustomButtonComponent.tsx

```diff
@@ -156,13 +157,14 @@ function useHandleServerActionMutation(lockTools: FormDataLockTools): UsePerform
   const instanceGuid = useNavigationParam('instanceGuid');
   const { handleClientActions, handleDataModelUpdate } = useHandleClientActions();
   const markNotReady = NodesInternal.useMarkNotReady();
+  const selectedLanguage = useCurrentLanguage();
 
   const { mutateAsync, isPending } = useMutation({
     mutationFn: async ({ action, buttonId }: PerformActionMutationProps) => {
       if (!instanceGuid || !partyId) {
         throw Error('Cannot perform action without partyId and instanceGuid');
       }
-      return doPerformAction(partyId, instanceGuid, { action: action.id, buttonId });
+      return doPerformAction(partyId, instanceGuid, { action: action.id, buttonId }, selectedLanguage);
```

Comment:
If it's a bug when this is not passed to the `doPerformAction()` function, maybe we should make it required in the function signature as well.